.github/workflows/ci.yml

```diff
@@ -214,7 +214,7 @@ jobs:
       run: npx vsce publish --pat ${{ secrets.MARKETPLACE_TOKEN }} ${{ steps.tag_name.outputs.tag }} --no-git-tag-version
 
     - name: Publish LSP to NPM
-      if: startsWith(github.ref, 'refs/tags/')
+      if: ${{ startsWith(github.event.head_commit.message, 'publish lsp') && (github.ref == 'refs/heads/master') }}
```

Comment:
Is this really needed? I thought it was nice that the LSP package is published at the same time as a stable VSCode extension release.

Reply:
Removed